resolvers.js

```diff
@@ -332,7 +336,7 @@ const Resolvers = {
       const {iso_image_ids, centroid_mzs, total_iso_ints, min_iso_ints, max_iso_ints} = hit._source;
       return centroid_mzs.map(function(mz, i) {
         return {
-          url: iso_image_ids[i] !== null ? config.img_upload.categories.iso_image.path + iso_image_ids[i] : null,
+          url: iso_image_ids[i] !== null ? `/${hit._source.ds_ion_img_storage}${config.img_upload.categories.iso_image.path}${iso_image_ids[i]}` : null,
```

Comment:
here an error appears: `iso_image` category contains sub-categories, which are not known at this stage

Reply:
is it a question to me?

Reply:
not a question, but rather an issue that I should not forget to address later on. I was just testing the `lcms-support` branch locally, and realized it's not working properly because of this line.

Reply:
Addressed this in another pull request